browser/conversion-tests/conversion_test.ts

```diff
@@ -81,16 +80,30 @@ window.addEventListener(loadEventName, () => {
 // Start compiling the WebAssembly as soon as the script is loaded. This lets
 // us initialize as quickly as possible.
 const go = new Go();
-WebAssembly.instantiate(wasmBuffer, go.importObject)
-    .then(module => {
-        go.run(module.instance);
+fetch('conversion_test.wasm')
+    .then(wasm => {
+        WebAssembly.instantiateStreaming(wasm, go.importObject)
```

Comment:
The [Go WebAssembly Wiki](https://github.com/golang/go/wiki/WebAssembly#getting-started) suggests using `instantiateStreaming` this way:

```suggestion
        WebAssembly.instantiateStreaming(fetch('conversion_test.wasm'), go.importObject)
```

In addition to being less code, it might also perform better.

Reply:
I see. That explains why `instantiateStreaming` can take either `Response` or `Promise<Response>` as its first argument.